docker/lr-guppy/Dockerfile

```diff
@@ -6,4 +6,14 @@ RUN apt-get install -y wget lsb-release
 RUN wget -O- https://mirror.oxfordnanoportal.com/apt/ont-repo.pub | apt-key add -
 RUN echo "deb http://mirror.oxfordnanoportal.com/apt xenial-stable non-free" | tee /etc/apt/sources.list.d/nanoporetech.sources.list
 RUN apt-get update
-RUN apt-get install -yq ont-guppy=5.0.16-1~xenial -V --no-install-recommends
+RUN apt-get install -yq ont-guppy=6.0.1-1~xenial -V --no-install-recommends
+
+RUN apt-get install -yq software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update
+RUN apt-get install -yq python3.6 curl && \
```

Comment:
does it need to be 3.6? I'd say we use 3.9 when possible.

Reply:
I had a hard time getting a newer version of python installed for use with the guppy_duplex tool.  Python 3.6 successfully installed and satisfied guppy_duplex's requirements, so I called it a win :).

Reply:
as long as the tool demands, we have no choice....